src/main/proto/cosmas.proto

```diff
@@ -117,15 +117,18 @@ message DeletePatchResponse {
     bytes content = 1;
 }
 
-message DeleteFileRequest {
+message DeleteFilesRequest {

```

Comment:
Let's pretend that we're going to update production microservice (which is kinda true)! 
If we want to replace a method in the API with newer one, we need to keep old method for a while, so that older clients could use older method while newer client use the new version. In our reality "older clients" is our current Papeeria and newer is one which will be rolled out after these changes.

Reply:
done